app/WebParsing/ReqParser.hs

```diff
@@ -425,6 +430,30 @@ courseAsModParser = do
     req <- andParser courseParser
     return $ REQUIREMENT req
 
+-- | Parses a literal "course" or "courses"
+courseLiteralParser :: Parser String
+courseLiteralParser = Parsec.spaces
```

Comment:
Don't include the spaces here. It's fine if the spaces are parsed as part of the `dept` below, you can strip them out when creating the `DEPARTMENT` value